backend/Dockerfile

```diff
@@ -2,11 +2,14 @@ FROM node:16-bullseye-slim
 
 WORKDIR /app
 
-COPY package*.json .
+COPY package.json package-lock.json ./
 
-RUN npm install
+RUN npm ci --only-production
 
 COPY . .
 
-CMD ["npm", "run", "start"]
+HEALTHCHECK --interval=10s --timeout=3s --start-period=10s \  
```

Comment:
I wonder if this will get rate limited because of this https://github.com/Infisical/infisical/blob/4d0969fbc32a759e469745e621652cd85f0eda1d/backend/src/index.ts#L65 🧐

Reply:
Can we remove rate limit for a given endpoint @dangtony98

Reply:
Oh, good catch! I'll look into this, if this path is also rate limited.

Reply:
Added skip in rate-limiter